examples/flash/main.py

```diff
@@ -24,13 +26,34 @@
 class FlashInterface(BoxLayout):
 
     def turn_on(self):
-        flash.on()
+        try:
+            flash.on()
+        except NotImplementedError:
+            self.ErMsg = "Feature under development for this platform!"
+            popup = Popup(title="Error!",
+                          content=Label(text=self.ErMsg),
+                          size_hint=(None, None), size=(350, 350))
+            popup.open()
 
     def turn_off(self):
-        flash.off()
+        try:
+            flash.off()
+        except NotImplementedError:
+            self.ErMsg = "Feature under development for this platform!"
+            popup = Popup(title="Error!",
+                          content=Label(text=self.ErMsg),
+                          size_hint=(None, None), size=(350, 350))
+            popup.open()
 
     def release(self):
-        flash.release()
+        try:
+            flash.release()
+        except NotImplementedError:
+            self.ErMsg = "Feature under development for this platform!"
+            popup = Popup(title="Error!",
+                          content=Label(text=self.ErMsg),
+                          size_hint=(None, None), size=(350, 350))
+            popup.open()
```

Comment:
Could these be done using one method instead of doing it 3 time?
and maybe if self.ErMsg says "Feature not available/implemented for this platform." that would make more sense?